practice/ttt/matrix_operations.clj

```diff
@@ -47,3 +47,11 @@
     (transpose-matrix matrix)
     (get-diagonals-of-matrix matrix)))
 
+
+(defn matrix->string
+  [matrix]
+  (apply str (for [row matrix]
+               (str
+                 (apply str "\n"
+                        (interpose " " row))))))
```

Comment:
nitpick: Using a thread last macro might make the section more readable:
```clojure
(->>  row
     (interpose " ")
     (apply str "\n")
     str)))
```
Also, do we need to `str` at the end if we're doing `apply str` in the previous step?